neurokit2/hrv/hrv_time.py

```diff
@@ -127,10 +127,13 @@ def hrv_time(peaks, sampling_rate=1000, show=False, **kwargs):
     out["pNN20"] = nn20 / len(rri) * 100
 
     # Geometrical domain
-    if "binsize" in kwargs.keys():
-        binsize = kwargs["binsize"]
-    else:
-        binsize = (1 / 128) * 1000
+    # if "binsize" in kwargs.keys():
+    # binsize = kwargs["binsize"]
+    # else:
+    # binsize = (1 / 128) * 1000
+    # consider using dict.get()
```

Comment:
interesting trick, didn't know that. @anshu-97 can you delete the commented code too since tests seem to pass?